servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ServerRespondsOnClosingTest.java

```diff
@@ -252,6 +251,8 @@ private void verifyResponse(String requestPath) {
         assertThat("Unexpected response meta-data", metaData.toString(US_ASCII), containsString(requestPath));
         ByteBuf payloadBody = channel.readOutbound();
         assertThat("Unexpected response payload body", payloadBody.toString(US_ASCII), equalTo(RESPONSE_PAYLOAD_BODY));
+        metaData.release();
+        payloadBody.release();
```

Comment:
I think these should be in a `finally` block bcz if assertions throw, your new lines won't be invoked